### ISSUE TITLE
Issue running Tensorflow Hub models (results differ from python)

### ISSUE THREAD

**Issue by andthenwhat** [serizba/cppflow#17]
I've fine-tuned a few models (inception v3, mobilenet-v2-140-224x224, resnet-v2-50) and attempted to run them using cppflow, but am receiving incorrect values in the output tensor (different than when running python).  

I retrained using a retrain.py modification for multi-label classification (https://github.com/isobar-us/multilabel-image-classification-tensorflow/tree/master/tf-multi-label-detection) and this pretrained model (https://tfhub.dev/google/imagenet/inception_v3/feature_vector/3), which I made sure wasn't  a TF2 module / model version.

On the c++ side, I'm loading and executing the model (code is inefficient, but it should be producing the float32-normalized, planar image output tensorflow is expecting):

```
static const int IMAGE_DIM = 299; // for convenience in this code example
Model model("inception-v3/output_graph.pb");
auto inpName = new Tensor(model, "Placeholder");
auto outName = new Tensor(model, "final_result");

cv::Mat img = imread("input.png"); // 3-channel RGB input
cv::Mat inp;

// resize and convert to float
cv::resize(img, inp, cv::Size(IMAGE_DIM, IMAGE_DIM));
inp.convertTo(inp2, CV_32F, 1.0/255.0);
cv::Mat planes[3];
cv::split(inp2, planes);

std::vector<float > img_data;

// write R, G, and B planes into img_data (input planes are BGR)
for (int channel = 2; channel >= 0; channel--) {
    for (int ii=0; ii < IMAGE_DIM*IMAGE_DIM; ii++) {
        img_data.push_back(planes[channel].at<float>(ii));
    }
}
inpName->set_data(img_data, {1, IMAGE_DIM, IMAGE_DIM, 3});
model.run(inpName, outName);

std::vector<float> predictions = outName->get_data<float>();

```
The output in predictions is wildly different than running the same image through the python label_image, e.g.:

```
python label_image.py --input_height=299 --input_width=299 --graph=./inception-v3/output_graph.pb --labels=labels.txt --input_layer=Placeholder --output_layer=final_result --image input.png
```

I'm happy to provide an end-to-end example if that's more beneficial, but before doing that: has anyone experienced a similar issue?  Am I missing something in the above code?  Any help is much appreciated!

**Comment by serizba** [comment]
Hi!

Sorry for the late answer. Getting different results from the Python version may be related to actually feeding different input data to the network. Have you checked that the data that you provide as input is in effect the same as in Python (you can try for example to print some values of the img_data and the values of the input in python and compare). 

If the values are exactly the same, could you provide an example I can reproduce?

Thanks

**Comment by serizba** [comment]
I proceed to close the issue due to lack of activity
